cpp/modmesh/onedim/pymod/wrap_onedim.cpp

```diff
@@ -128,9 +128,9 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapEuler1DCore
 
         (*this)
             .def_timed("update_cfl", &wrapped_type::update_cfl, py::arg("odd_plane"))
-            .def_timed("march_half_so0", &wrapped_type::march_half_so0, py::arg("odd_plane"))
             .def_timed("treat_boundary_so0", &wrapped_type::treat_boundary_so0)
             .def_timed("treat_boundary_so1", &wrapped_type::treat_boundary_so1)
+            .def_timed("march_half_so0", &wrapped_type::march_half_so0, py::arg("odd_plane"))
```

Comment:
This is not a refactoring PR.  It is unnecessary to change the order of existing code.  Next time please avoid doing it in a non-refactoring PR.